snap/snapcraft.yaml

```diff
@@ -4,24 +4,49 @@ version: 1.76.6
 license: BSD-3-Clause
 grade: stable
 confinement: strict
+contact: solutions-engineering@lists.canonical.com
```

Comment:
The source-code label is missing as well

Reply:
Thanks! Added. :)